README.md

```diff
@@ -1824,4 +1824,134 @@ When you are dealing with exceptions in your code, use `throw new RuntimeExcepti
 
 - It gives the programmer more control over the exception.
 - It gives the programmer the power of logging exceptions.
-- It reduces visibility of the working of your code for potential hackers.
\ No newline at end of file
+- It reduces visibility of the working of your code for potential hackers.
+
+## Unix Directories
+
+For operating systems built on Unix (e.g. Linux and Mac) the highest level of the directory, also called the root folder, is denoted by `/`.
+
+The home folder can be written as `/home/username` (e.g.`/home/jen`) or `~`.
+
+The command `pwd` (print working directory) will print the current directory to the screen.
```

Comment:
Could you give an example of this? If you're in home (`~`) it won't simply print `~`.